pkg/features/kube_features.go

```diff
@@ -862,6 +862,17 @@ const (
 	//
 	// Enables specifying resources at pod-level.
 	PodLevelResources featuregate.Feature = "PodLevelResources"
+
+	// owner: @ffromani
+	// beta: v1.33
+	//
+	// Disable CPU Quota for containers which have exclusive CPUs allocated.
```

Comment:
Make sure the add "temporary allow to disable" wording. It MUST be clear that this is something that will not be allowed soon.

Reply:
added!